features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/BitmapPool.kt

```diff
@@ -103,6 +103,21 @@ internal class BitmapPool(
         }?.apply { markBitmapAsUsed(this) }
     }
 
+    override fun onConfigurationChanged(newConfig: Configuration) {}
+
+    @Synchronized
```

Comment:
are you sure it is caused by these methods ? Were you able to reproduce it locally ?

Reply:
yes, I reproduced it with the test for onLowMemory. For onTrimMemory it's would probably also occur, but since the tests don't have access to the backing cache and have to call clear - clear is already synchronized so it prevents the issue from being visible.